### PR TITLE
06-20 Dev test contract deployment

### PR DESCRIPTION
**Pull request by lindsgil**
## Description of the change

Dev test contract deployment configuration

- [x] Queue deployments to multisig
- [x] Approve PR
- [x] Execute deployment tx
- [x] Run post-deployment script

### CODE REVIEW

**Review by ryley-o — APPROVED** [review]
lgtm - suggested a few changes 👍

File: packages/contracts/deployments/engine/V3/internal-testing/dev-engine-base-mainnet/2024-06-20-deployment-config.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    // common values include `studio`, `fullyonchain`, `flex`, or partnerships like `artblocksxpace`.
+    // also note that if you desire to create a new vertical, you will need to add the vertical name to the
+    // `project_verticals` table in the database before running this deploy script.
+    defaultVerticalName: "unassigned",
+  },
+  {
+    // 0 for Engine, 1 for Engine Flex
+    engineCoreContractType: 1,
+    tokenName: "DEV TEST | 0",
+    tokenTicker: "DEV_TEST_0",
```
[the remark this answers is not in the document]
Comment (lindsgil) [reply packages/contracts/deployments/engine/V3/internal-testing/dev-engine-base-mainnet/2024-06-20-deployment-config.ts:50]: 🙏

File: packages/contracts/deployments/engine/V3/internal-testing/dev-engine-base-mainnet/2024-06-20-deployment-config.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  transactionHash:
+    "",
+};
+
+export const deployConfigDetailsArray = [
+  {
+    // 0 for Engine, 1 for Engine Flex
+    engineCoreContractType: 0,
+    tokenName: "DEV TEST | 0",
+    tokenTicker: "DEV_TEST_0",
```
[the remark this answers is not in the document]
Comment (lindsgil) [reply packages/contracts/deployments/engine/V3/internal-testing/dev-engine-base-mainnet/2024-06-20-deployment-config.ts:23]: 🙏

File: packages/contracts/deployments/engine/V3/internal-testing/dev-engine-base-mainnet/2024-06-20-deployment-config.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    engineCoreContractType: 1,
+    tokenName: "DEV TEST | 0",
+    tokenTicker: "DEV_TEST_0",
+    // see /scripts/util/constants.ts::MAIN_CONFIG for the correct address if prod deployment
+    renderProviderAddress: "0xc8D1099702cB95baf954a4E3e2bEaF883314f464",
+    platformProviderAddress: "0x0000000000000000000000000000000000000000",
+    // set to the address you want to transfer the superAdmin role to
+    // (can be the zero address if you have an existing Admin ACL Contract and are not transferring)
+    // dev admin address
+    newSuperAdminAddress: "0x3c6412FEE019f5c50d6F03Aa6F5045d99d9748c4",
```
[the remark this answers is not in the document]
Comment (lindsgil) [reply packages/contracts/deployments/engine/V3/internal-testing/dev-engine-base-mainnet/2024-06-20-deployment-config.ts]: Updated!

File: packages/contracts/deployments/engine/V3/internal-testing/dev-engine-base-mainnet/2024-06-20-deployment-config.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+// It is intended to be imported by the batch engine factory deployer by running `deploy:v3-engine:mainnet`, `deploy:v3-engine:staging` or `deploy:v3-engine:dev`.
+
+export const deployNetworkConfiguration = {
+  network: "base",
+  // environment is only used for metadata purposes, and is not used in the deployment process
+  // Please set to "dev", "staging", or "mainnet", arbitrum as appropriate
+  environment: "base",
+  useLedgerSigner: true,
+  useGnosisSafe: true,
+  safeAddress: "0x437461553625d179A49c265258Fa71E442bBffEF",
```
[the remark this answers is not in the document]
Comment (lindsgil) [reply packages/contracts/deployments/engine/V3/internal-testing/dev-engine-base-mainnet/2024-06-20-deployment-config.ts]: Good call! Updated

File: packages/contracts/deployments/engine/V3/internal-testing/dev-engine-base-mainnet/2024-06-20-deployment-config.ts [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  {
+    // 0 for Engine, 1 for Engine Flex
+    engineCoreContractType: 0,
+    tokenName: "DEV TEST | 0",
+    tokenTicker: "DEV_TEST_0",
+    // see /scripts/util/constants.ts::MAIN_CONFIG for the correct address if prod deployment
+    renderProviderAddress: "0xc8D1099702cB95baf954a4E3e2bEaF883314f464",
+    platformProviderAddress: "0x0000000000000000000000000000000000000000",
+    // dev admin address
+    newSuperAdminAddress: "0x3c6412FEE019f5c50d6F03Aa6F5045d99d9748c4",
```
[the remark this answers is not in the document]
Comment (lindsgil) [reply packages/contracts/deployments/engine/V3/internal-testing/dev-engine-base-mainnet/2024-06-20-deployment-config.ts]: 🙏 Updated!

**Review by ryley-o — APPROVED** [review]

**Pull request merged**